### ISSUE TITLE
AttributeError: 'SES' object has no attribute 'close'

### ISSUE THREAD

**Issue by bbaker6225** [django-ses/django-ses#186]
In upgrading `django-ses` to version `1.0.0`, we began seeing the following error when using `mail.get_connection` to send emails:
```python
AttributeError: 'SES' object has no attribute 'close'.
```
Here is the link to the code that uses this and ran into the issue: https://github.com/edx/license-manager/blob/aee7af851ddf87d9a2ba669693d0a9b995e51cb2/license_manager/apps/subscriptions/emails.py#L91-L102

The error doesn't seem to be having any effect on actual email sending as we are still seeing successful email deliveries from both the above code and django-ses email code in https://github.com/edx/edx-platform , but the exception is causing server errors in places that have not yet been wrapped with a try/except.

Is it possible to get a fix for this issue out soon? Thanks in advance

**Comment by pcraciunoiu** [comment]
@GitRon want to look at this? Or @bbaker6225 if you want to submit a fix and verify it, that'd be great.

**Comment by pcraciunoiu** [comment]
@bbaker6225 try installing this version from github and let me know if it fixes the issue. I don't have a real project handy to try it, so would be good if you can confirm before I make a release:

```sh
pip install git+https://github.com/django-ses/django-ses.git@3069b88dc32d7928c606c783c40119f0cd07633e
```

**Comment by bbaker6225** [comment]
@pcraciunoiu yep, I'll test this out now. Thanks for drafting that

**Comment by bbaker6225** [comment]
@pcraciunoiu that looks like it worked for us, so I think it's good for release. The only caveat is we do have general exception handling for the rest of our emailing, so there is the possibility it's raising some exception other than `AttributeError`

**Comment by pcraciunoiu** [comment]
@bbaker6225 So you confirmed it doesn't raise this `object has no attribute 'close'` error anymore right?

If it did raise this or other errors, you would have a log of it?

**Comment by bbaker6225** [comment]
@pcraciunoiu We're currently running into some logging issues for license-manager so we're not certain there are no other exceptions being raised. I see you released the changes in #187 so I'll try and test those out
